modules/ROOT/pages/customize.adoc

```diff
@@ -1,101 +1,177 @@
-= Customize the Look and Feel of Your API Community
+= API Community Look and Feel Customization
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Use the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] to customize the branding, look and feel, and site structure of your community pages to match your desired brand identity.
+The https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] enables
+you to create a community with your desired brand identity, look and feel, and
+site structure by providing control over the community's components, themes,
+pages, and its general, theme, and SEO settings.
 
-Select *Community Builder* in the API Community Manager (ACM) Control Panel. By default, the
-Community Builder displays the *Home* page and the Pages drop-down list displays *Home*. You can select
-the Pages drop-down list to display a list of community pages and specify a different page for customization.
+In the API Community Manager (ACM) Control Panel, the *Community Builder* shows
+your site for editing. The builder is where you edit each page in your
+community, create new pages, and add and edit the components in each page.
 
-The *Community Builder* interface contains the customization icons described here.
+The *Community Builder* interface contains the elements described here.
+
+== Pages List
+
+The *Pages* list shows the pages included with ACM and the pages you have added,
+and enables you to select and edit each page. The included page *Home* is
+selected by default.
+
+The *Pages* list enables you to create new pages with the *New Page* option.
+
+A new page can be an *Object Page* or *Standard Page*. An *Object Page* is used
```

Comment:
```suggestion
A new page can be an object page or a standard page. An object page is used
```
In this context, these are types of pages and not references to their appearance in the UI.